src/components/assets/Assets.vue

```diff
@@ -28,21 +28,23 @@
       </div>
     </div>
     <Teleport to="#app--main">
-      <ModalXcmTransfer
-        :is-modal-xcm-transfer="isModalXcmTransfer"
-        :handle-modal-xcm-transfer="handleModalXcmTransfer"
-        :handle-update-xcm-token-balances="handleUpdateXcmTokenAssets"
-        :account-data="accountData"
-        :token="token"
-      />
-
-      <ModalXcmBridge
-        :is-modal-xcm-bridge="isModalXcmBridge"
-        :handle-modal-xcm-bridge="handleModalXcmBridge"
-        :account-data="accountData"
-        :token="token"
-        :handle-update-xcm-token-balances="handleUpdateXcmTokenAssets"
-      />
+      <div :class="!isLoading && 'highest-z-index'">
```

Comment:
Correct me if I'm wrong, but it seems easier to change the modal style to have the highest z index rather than manually coding this for all components.

Reply:
It is so tricky because we have to consider the `z-index` of the three elements, `Header`, `Modals`, and `Loading animation` under different situations. The `z-index` priority must change dynamically.

We cannot make `Modals` always located at the highest position because the `Loading animation` will be hidden by modals. We also cannot make the `Loading animation` at the highest position because we might want to change the network when the APP is connecting to an endpoint. We also cannot make the `Header` at the highest `z-index` position as modals will be hidden by `Header`.

Z-index priority:
When the APP is connecting an endpoint : Header > Loading 
Modal is opened (Before starting loading): Modal > Header
Modal is opened (After starting loading):    Loading > Modal 

<img width="1166" alt="image" src="https://user-images.githubusercontent.com/92044428/181905224-0c94cf21-4a16-412b-b943-2e54c64fb933.png">

Reply:
Perhaps I can create a wrapper for modals. Let me try!

Reply:
I've fixed in this commit. https://github.com/AstarNetwork/astar-apps/pull/477/commits/0e1794181bb574e01c5f785df8f74e7d9d20b2ea
I think this commit makes more sense.